lib/jekyll-import/importers/enki.rb

```diff
@@ -52,10 +52,10 @@ def self.process(options)
                              :encoding => "utf8")
 
         db[SQL].each do |post|
-          name = [ format("%.04d", post[:date].year),
-                   format("%.02d", post[:date].month),
-                   format("%.02d", post[:date].day),
-                   post[:slug].strip, ].join("-")
+          name = [format("%.04d", post[:date].year),
+                  format("%.02d", post[:date].month),
+                  format("%.02d", post[:date].day),
+                  post[:slug].strip,].join("-")
```

Comment:
I generally do 

```ruby
name = [
  ...,
  ..., 
].join("-")
```

Generally works well for me. You can omit the trailing comma if rubocop says too, I forget our convention.